math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -2198,14 +2198,14 @@ RVec<T> Take(const RVec<T> &v, const int n)
 template <typename T>
 RVec<T> Drop(const RVec<T> &v, RVec<typename RVec<T>::size_type> idxs)
 {
-   RVec<T> r;
-   if (v.size() > idxs.size())
-      r.reserve(v.size() - idxs.size());
-
    // clean up input indices
    std::sort(idxs.begin(), idxs.end());
    idxs.erase(std::unique(idxs.begin(), idxs.end()), idxs.end());
 
+   RVec<T> r;
+   if (v.size() > idxs.size())
```

Comment:
This branch is likely taken; perhaps we could also use `R__likely(...)`.

Reply:
should be ok :) thank you all for the nice reviews!